icon-key-concepts/parallel-processing.md

```diff
@@ -3,86 +3,63 @@ title: "Parallel Processing"
 excerpt: "Executing transactions in parallel"
 ---
 
-Performance of blockchain can be influenced by various factors.
-One of the major factors is the execution time of transactions.
-To reduce this, we may apply the parallel processing on it.
+Various factors affect the blockchain performance. One of the major factors is the **execution time** of transactions. In this article, we illustrate our R&D approach to increase the blockchain performance by reducing the transaction execution time. 
 
+## Transaction Execution Time 
+Transactons Per Second (TPS) is the number of transactions that a network can process during a single second. TPS is a key metric of blockchain performance. 
+Transaction can be defined from the submission to the network until it is included in the block. In ICON, the number of transactions included in a block are always quaranteed to be executed within one block generation time, so, it is a good representation of transaction throughput, and proportional to the TPS.  
 
-## Problem definition
+TPS is mainly affected by following factors.
+* Travel time of transaction to the consensus node 
+* Communication time among consensus nodes
+* Execution time of transaction 
 
-Performance of blockchain is influenced by following major factors.
-* Execution time of transactions
-* Transfer time of transactions
-* Communication time for consensus
+Travel and communication time are network bound, and there has been continuous attempt to optimize the network protocol in blockchain. Further optimizing the network protocol is not trivial as it invloves the risk of breaking the liveness and safety properties in the blockchain network. 
+On the other hand, reducing the execution time has not been touched yet. If we reduce the transaction execution time, we can improve the performance of blockchain without risking the network stability.
 
-Various blockchains have already improved their protocols to reduce network
-traffics. Considering liveness and safety of blockchain, it's hard to reduce
-network traffics more without risk.
+## Parallel Execution
 
-If we can reduce execution time, we may improve the performance of blockchain
-without risk.
+The most straightforward approach to reduce the transaction execution time is using stronger computing power. This is called scale-up or vertical scaling. However, increasing computing power is expensive and has physical limitation. Instead, in modern computing, increasging overall throughput with the parallel execution using multiple cores is more common approach. 
 
+That said, in blockchain, sequential execution of transactions has been the norm, because, blockchain network represents a world state - a collection of accounts and their states, and every consensus node must agree on the state change after a transaction. Because previous transctions can affect the execution of the current transaction, it is safe to execute transactions sequentially.
```

Comment:
```suggestion
That said, in blockchain, sequential execution of transactions has been the norm, because, blockchain network represents a world state - a collection of accounts and their states, and every consensus node must agree on the state change after a transaction. Because previous transactions can affect the execution of the current transaction, it is safe to execute transactions sequentially.
```